src/api/airtableApi.jsx

```diff
@@ -94,3 +94,28 @@ export const create = async newRecord => {
     console.log(error);
   }
 };
+
+export const updateRecords = async titles => {
+  try {
+    const response = await fetch(tableUrl, {
+      body: JSON.stringify({
+        records: titles.map(title => {
+          const { id: _, ...titleWithoutId } = title;
+          return {
+            id: title.id,
+            fields: titleWithoutId,
+          };
+        }),
+      }),
+      headers: {
+        ...defaultHeaders,
+        'Content-Type': 'application/json',
+      },
+      method: 'PATCH',
```

Comment:
PATCH will only update fields that are in the request and will not overwrite any existing fields in a record not contained in the request. A PUT request would overwrite the entire updated record, dropping existing fields if they are not in the update request.